src/BlockEntities/ChestEntity.h

```diff
@@ -62,11 +69,9 @@ class cChestEntity :
 	/** Opens a new chest window where this is the primary chest and any neighbour is the secondary. */
 	void OpenNewWindow();
```

Comment:
This comment could be clarified - specifying that this is only _allowed_ in that condition, and that chest order will not somehow be switched around.